src/toil/fileStore.py

```diff
@@ -1843,13 +1843,9 @@ def __new__(cls, fileStoreID, *args):
         return super(FileID, cls).__new__(cls, fileStoreID)
```

Comment:
IMO you should either change all `super`s to the new format (using `from builtins import super`) or none. Right now it's basically half and half.

Reply:
I'm in the process of converting them all.  Have not completed this yet though.  If you feel I should convert all of them before this PR is finished, I'll go in and get the rest to work.

Reply:
Nah, it's cool, as long as there's a plan to convert them eventually.